### ISSUE TITLE
XAnalyzer - The Code is changed, but does not change in the debugger.

### ISSUE THREAD

**Issue by ffrlx** [ThunderCls/xAnalyzer#46]
Hi!

XAnalyzer is not updating the function parameters, even with automatic analysis on. If I modify a string, the debugger x32dbg, apparently, takes the previous code that is in the DB folder. All cleaning options are on. Is there a solution? Is it possible for XAnalyzer to recognize all ASCII table characters? For instance: recognizing strings like 0x82, 0xA0, etc. I'm using the latest version of x64dbg and xAnalyzer.

Thanks.

**Comment by ThunderCls** [comment]
I'm sorry but I don't quite understand your statement. Are you referring to the debugger or the plugin? Can you elaborate? You can use screenshots if you want.

**Comment by ffrlx** [comment]
![Debugger and xAnalyzer](https://user-images.githubusercontent.com/41217980/81130359-e2326b00-8f1d-11ea-9026-9c5f05689fcd.png)

Sorry, english it's not my native language.  
About the plugin. Look image.

**Comment by ffrlx** [comment]
When I do a new compilation, in debbuger the hexdump is correct, but in disassembler it is not.

**Comment by ThunderCls** [comment]
New changes done to the binary are not automatically reflected by the plugin in the function comments, you need to manually launch a new analysis so the data can be updated to the newest. In this case, for instance, since you're only modifying one specific function params, you could just select every argument and perform a "selection" analysis with the plugin, values should update just fine

**Comment by ffrlx** [comment]
Okay, this worked. Any chance to implement for words with special characters (for instance: á, é, í, ó, etc.)? Words with these characters do not appear in the debugger comments.

Thank you.

**Comment by ThunderCls** [comment]
Yeah so far it doesn't support those, but it will in the next major version.
Thanks for reporting!
